src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, serif;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), font-families should be listed with fallbacks. It's a good practice to include a generic font family as a fallback. In this case, 'sans-serif' should be used instead of 'serif' as 'Roboto' is a sans-serif font.